dspace/config/spring/api/external-services.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:util="http://www.springframework.org/schema/util"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd"
+    default-lazy-init="true">
+
+    <bean class="org.dspace.external.service.impl.ExternalDataServiceImpl"/>
+
+    <bean class="org.dspace.external.provider.impl.SherpaJournalDataProvider" init-method="init">
+        <property name="sourceIdentifier" value="sherpaJournal"/>
+        <property name="url" value="http://www.sherpa.ac.uk/romeo/api29.php"/>
+        <!-- please register for a free api access key to get many benefits -->
+        <property name="apiKey" value=""/>
```

Comment:
As noted above all `<property>` settings in this XML file should be pulling in values from dspace.cfg.  Otherwise we are making these settings more difficult to edit/change (as DSpace installers couldn't easily override them in a local.cfg, unless they are in dspace.cfg)